Traits/ResponseTrait.php

```diff
@@ -39,12 +40,22 @@ public function transform(
         array $meta = [],
         $resourceKey = null
     ) {
-        // create instance of the transformer
-        $transformer = new $transformerName;
 
+        // if string was passed, then create instance of the transformer
+        if(is_string($transformerName)) {
+            $transformer = new $transformerName;
```

Comment:
@johannesschobel this is declaring a `instance`, but when a `transformer` has a argument on `__construct`, it will return an exception of `Too few arguments`